docs/explanations/flyscanning.md

```diff
@@ -0,0 +1,86 @@
+# Flyscanning
+
+We can classify data collections across a spatial or parameterised domain into two broad categories: step scanning, and flyscanning.
+
+* Step scanning: The system performs a sequence of discrete operations: move to the next position, settle, acquire data, repeat.
+This ensures well-defined data points but may result in longer total acquisition times.
+Bluesky plans specify this type of measurement through devices' [Movable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Movable),
```

Comment:
We have intersphinx on this project, so it can pick up python class links without needing the full URL:
```suggestion
Bluesky plans specify this type of measurement through devices' [Movable](#bluesky.protocols.Movable),
```

Reply:
Weird and done